ck3-tiger.conf

```diff
@@ -13,21 +13,66 @@ languages = {
 	# If you don't specify anything in languages, the validator will check all languages.
 }
 
-# There can be multiple ignore sections.
-ignore = {
-	# This will ignore the specified warnings.
+# The ignore blocks still work but are deprecated. It might be removed in a later version.
+# Consider using the filter.blacklist block instead.
+ignore = { }
 
-	# Each key entry specifies a key to ignore.
-	# The key names are in the output, in parentheses after ERROR or WARNING.
-	# If there are no keys, all errors in the specified files will be ignored.
-	key = brace-placement
+# Allows configuring a number of ways to reduce the number of reports that are printed by CK3-Tiger.
+filter = {
 
-	# Each file entry specifies a file in which to ignore the specified key or keys.
-	# If there are no file entries, the keys are ignored in all files.
-	file = events/POD_initialization/POD_maintenance_events.txt
+    # Reports that aren't important enough will not be printed.
+    # Valid values are: Error, Warning, Info, Untidy.
```

Comment:
I've been thinking about those, and it should maybe be

Error, Warning, Untidy, Tips

"Info" is currently used only by advice-level reports and those are used for tips: things that could be improved but aren't wrong. I hope to expand that section actually, especially regarding performance tips.

I think they should be below "untidy" because untidy things are slightly wrong, while tips are about things that are not wrong at all. The Rust equivalent would be clippy suggesting more idiomatic ways to write code.

I've also been using advice for some Untidy things but I can change those when the new system is in.